WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -490,28 +491,33 @@ class MySiteViewModel @Inject constructor(
                 )
         )
 
+        val jetpackBadge = JetpackBadge.takeUnless { buildConfigWrapper.isJetpackApp }
+
         return mapOf(
                 MySiteTabType.ALL to orderForDisplay(
                         infoItem,
                         cardsResult,
                         dynamicCards,
-                        siteItems
+                        siteItems,
+                        jetpackBadge
                 ),
                 MySiteTabType.SITE_MENU to orderForDisplay(
                         infoItem,
                         cardsResult.filterNot {
                             getCardTypeExclusionFiltersForTab(MySiteTabType.SITE_MENU).contains(it.type)
                         },
                         if (shouldIncludeDynamicCards(MySiteTabType.SITE_MENU)) dynamicCards else listOf(),
-                        siteItems
+                        siteItems,
+                        jetpackBadge
```

Comment:
As per [discussion on the channel](p1657660798346659-slack-C03MHJFF58A).  It may not be required here on Menu tab

Reply:
Also, need to update MySiteViewModelTest to make it pass build

Reply:
> Also, need to update MySiteViewModelTest to make it pass build
Fixed this, I adapted the failing test and added 2 new unit tests for the Jetpack badge visibility 🎉

Reply:
> As per [discussion on the channel](https://github.com/wordpress-mobile/WordPress-Android/pull/p1657660798346659-slack-C03MHJFF58A). It may not be required here on Menu tab

I [updated the code to **not** show the badge on the site menu](https://github.com/wordpress-mobile/WordPress-Android/pull/16901/commits/204d1917fd41cf28ff63ce023dd83d4abc1054c3), thank you for this remark and the review @ravishanker 🙇 🙏 🥇